lightspeed_stack_providers/providers/inline/agents/lightspeed_inline_agent/__init__.py

```diff
@@ -15,6 +17,7 @@ async def get_provider_impl(config: LightspeedAgentsImplConfig, deps: dict[Api,
         deps[Api.safety],
         deps[Api.tool_runtime],
         deps[Api.tool_groups],
+        policy,
```

Comment:
Is this policy bits part of the patch ? I don't see the policy parameter being used anywhere in the shield.

Or was it something that a newer version of llama-stack included that is a required parameter ?